shell/initialize/app-extended.js

```diff
@@ -1,49 +1,23 @@
 // Taken from @nuxt/vue-app/template/index.js
 // This file was generated during Nuxt migration
-
```

Comment:
I've renamed it from `index.js`, as this is not the starting point of the whole folder.